modules/ROOT/pages/shared-resources.adoc

```diff
@@ -20,6 +20,7 @@ To share the metadata, keep the Mule domain project open in Anypoint Studio. Oth
 
 * Anypoint Studio 7.x
 * Mule runtime engine 4.x
+* Run Mule runtime engine On-Premises
```

Comment:
Again, "On-Premises" isn't being used as an adjective, but as a prepositional phrase. So, remove the hyphen.